pkg/apis/experimental/types.go

```diff
@@ -639,3 +639,36 @@ type ClusterAutoscalerList struct {
 
 	Items []ClusterAutoscaler `json:"items"`
 }
+
+// Selector is a label query over a set of API objects. MatchLabels and
+// MatchExpressions are ANDed. An empty Selector matches all objects. A
+// null Selector matches no objects.
+type Selector struct {
+	// MatchLabels is a list of labels.
```

Comment:
This comment is not correct -- this is a map (I guess from label to value). Assuming that's what is intended here, what is the purpose of this? Can't these be expressed using MatchExpressions with operator "In"? For example

MatchLabels = {{color, red} {temperature, ot}}
can be expressed as
MatchExpressions = ["color" IN "red", "temperature" IN "hot"]